src/server.ml

```diff
@@ -272,8 +296,10 @@ let rec wait_loop process_params verbose accept =
 					) m.m_types;
 					if m.m_extra.m_kind <> MSub then Typeload.add_module ctx m p;
 					PMap.iter (Hashtbl.replace com2.resources) m.m_extra.m_binded_res;
-					PMap.iter (fun _ m2 -> add_modules (tabs ^ "  ") m0 m2) m.m_extra.m_deps);
+					if ctx.Typecore.in_macro || com2.display.dms_full_typing then
```

Comment:
If you wonder why I want to avoid this, here's what happens when you pull in all dependencies of haxe.EnumTools in an empty HaxeFlixel project: http://paste2.org/aNdKPMC7